functions.php

```diff
@@ -129,12 +132,40 @@ function ampconf_widgets_init() {
 }
 add_action( 'widgets_init', 'ampconf_widgets_init' );
 
+/**
+ * Filters the AMP custom css to include the CSS from out theme.
+ *
+ * @param string $css The combined custom css.
+ *
+ * @return string $css
+ */
+function ampconf_custom_styles( $css ) {
+	$path   = get_template_directory() . '/assets/dist/css/main.css';
+	$styles = file_get_contents( $path ); // phpcs:ignore WordPress.WP.AlternativeFunctions -- Not a remote file.
+
+	return $styles . $css;
+}
+add_filter( 'amp_custom_styles', 'ampconf_custom_styles' );
+
+/**
+ * Adds custom component scripts to the document.
+ *
+ * @param array $amp_scripts AMP Component scripts, mapping component names to component source URLs.
+ *
+ * @return array
+ */
+function ampconf_amp_component_scripts( $amp_scripts ) {
+	$amp_scripts['amp-form'] = 'https://cdn.ampproject.org/v0/amp-form-0.1.js';
```

Comment:
This should be automatically be included by the plugin once the document sanitization is completed. Since it isn't the case at the moment, add `// @todo` as a reminder to remove it.